Makefile

```diff
@@ -6,7 +6,7 @@ UNAME_S := $(shell uname -s)
 
 # If you're using docker and docker-compose, you can use this Makefile to run
 # commands in your docker images by setting the DOCKER_PREFIX environment variable
-# to: DOCKER_PREFIX="docker-compose run --rm web"
+# to: DOCKER_PREFIX="docker-compose run --rm web --"
```

Comment:
Also you might want to look at using a shell via exec for stuff like this. Back when all this was done exec didn't exist. Now you can jump into a shell on a running container which is much easier. See docker-utils [1] which provides a nice wrapper to docker-compose for doing stuff like `docker-utils bash [service]` [2]

[1] https://pypi.python.org/pypi/docker-utils
[2] https://github.com/andymckay/docker-utils#bash